src/main/java/plugily/projects/buildbattle/handlers/menu/registry/particles/ParticlesOption.java

```diff
@@ -46,11 +45,12 @@ public ParticlesOption(OptionsRegistry registry) {
       @Override
       public void onClick(InventoryClickEvent event) {
         event.getWhoClicked().closeInventory();
-        BaseArena arena = (BaseArena) registry.getPlugin().getArenaRegistry().getArena((Player) event.getWhoClicked());
-        if(arena == null) {
+
+        org.bukkit.entity.HumanEntity who = event.getWhoClicked();
```

Comment:
importing HumanEntity
if we are already here renaming who to player would be also nice / instanceof player check not to forget